<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="conceptSearchEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="conceptSearchLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Search Concept:</string>
   </property>
  </widget>
  <widget class="QLabel" name="tripleStoreLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TripleStore:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tripleStoreEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="findConcept">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>106</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Class</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="findProperty">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>106</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Property</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="searchResultLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Search Results</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>440</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QListWidget" name="searchResult">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>600</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="costumproperty">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="costumpropertyButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Own URI</string>
   </property>
  </widget>
  <widget class="QLabel" name="costumpropertyLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Define Own URI:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
